app/routes/index.js

```diff
@@ -1,10 +1,14 @@
 import HttpStatus from 'http-status'
 import db from '../../database/models'
+import userRoutes from './user'
```

Comment:
Lets change name from userRoutes to authRoutes

Reply:
renaming the routes to `auth` isn't a bad idea, but that means we only limit the routes to authentication only.  Just a thought, will the application at any given point need the functionality to fetch user(s)? let's say for example get the users who have responded to a particular event?